src/QualificationRunner.Core/Services/QualificationRunner.cs

```diff
@@ -161,6 +160,41 @@ private async Task<ObservedDataMapping> copyObservedData(ObservedDataMapping obs
          };
       }
 
+      private Task<Section[]> copySectionContents(Section[] sections)
+      {
+         if (sections == null)
+            return Task.FromResult(Array.Empty<Section>());
+
+         return Task.WhenAll(sections.Select(copySectionContent));
+      }
+
+      private async Task<Section> copySectionContent(Section section)
+      {
+         Task<string> downloadRemoteSectionContent() => downloadRemoteFile(section.Content, CONTENT_DOWNLOAD_FOLDER, "Content");
+
+         var contentAbsolutePath = absolutePathFrom(_runOptions.ConfigurationFolder, section.Content);
+
+         if (!localFileExists(contentAbsolutePath))
+            contentAbsolutePath = await downloadRemoteSectionContent();
+
+         if (!localFileExists(contentAbsolutePath))
+            throw new QualificationRunException(ContentFileNotFound(contentAbsolutePath));
+
+         var fileInfo = new FileInfo(contentAbsolutePath);
+
+         DirectoryHelper.CreateDirectory(_runOptions.ContentFolder);
+         var copiedContentDataFilePath = absolutePathFrom(_runOptions.ContentFolder, fileInfo.Name);

```

Comment:
If a file was defined with the same name as en existing one, it will colide here and the file will be overwritten

Reply:
OK for me. 
@sfrechen @incei FYI